chips/sam4l/src/scif.rs

```diff
@@ -333,7 +376,52 @@ pub unsafe fn setup_pll_osc_48mhz() {
     );
 
     // Wait for the PLL to become locked
-    while (*SCIF).pclksr.matches_all(Interrupt::PLL0LOCK::CLEAR) {}
+    while !(*SCIF).pclksr.is_set(Interrupt::PLL0LOCK){}
+}
+
+pub unsafe fn disable_pll() {
```

Comment:
Why are these marked `unsafe`?

Reply:
The existing code in scif.rs used unsafe functions instead of unsafe blocks so I continued using that structure in the code I added. In general, unsafe is needed since we dereference the SCIF pointer.